### PR TITLE
Manual move documentation

### PR DESCRIPTION
**Pull request by MahshadJ**
added documentation for the new feature: manual move generator. This feature allows the user to set moves during placement.

#### Description

The documentation explains the purpose of this new feature and how to use it. The issue with the images for the debugger documentation have also been fixed.

#### Motivation and Context

It guides the user in using the manual move generator

#### Types of changes

- [x] Bug fix (change which fixes an issue)
- [ ] New feature (change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

#### Checklist:

- [ ] My change requires a change to the documentation
- [x] I have updated the documentation accordingly
- [ ] I have added tests to cover my changes
- [ ] All new and existing tests passed

### CODE REVIEW

File: doc/src/vpr/debug_aids.rst [hunk 1]
```diff
@@ -103,3 +103,42 @@ Available Operators
...
+
+Selecting the block to move
+~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+You can choose the block you want to move by entering its block id or block name in the corresponding entry field, or you can simply click on that specific block. For better visualization, once you indicate your block through the entry field and press proceed, your block will be highlighted on the GUI.
+
+Selecting the to location
+~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The “to location” of your move consists of the x location, y location, and subtile. If you’re unsure of the subtile value input 0.
```
Comment (vaughnbetz) [inline doc/src/vpr/debug_aids.rst:144]: (add at end)  as most (x,y) locations in most devices have only a single subtile.

File: doc/src/vpr/debug_aids.rst [hunk 2]
```diff
@@ -103,3 +103,42 @@ Available Operators
...
+
+.. image:: https://www.verilogtorouting.org/img/manualMoveCheckbox.png
+    :align: center
+    
+Then after pressing the “Proceed” button a window pops up that asks for the block you want to move, and the location you would want to move it to. 
+
+.. image:: https://www.verilogtorouting.org/img/manualMoveWindow.png
+    :align: center
+    
+By pressing the “calculate costs” button you can see all the costs of your move, as well as the placer’s move_outcome and then decide to reject or accept the move.
```
Comment (vaughnbetz) [inline doc/src/vpr/debug_aids.rst:129]: Explain what placer's move outcome is.  Whether the placer would choose to accept or reject the move?  Can it also say the move is illegal/aborted?  Explain what this is.

File: doc/src/vpr/debug_aids.rst [hunk 3]
```diff
@@ -103,3 +103,42 @@ Available Operators
...
+How to use the manual move generator
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+First, you should make sure the manual move option is enabled, as seen in the figure.
+
+.. image:: https://www.verilogtorouting.org/img/manualMoveCheckbox.png
+    :align: center
+    
+Then after pressing the “Proceed” button a window pops up that asks for the block you want to move, and the location you would want to move it to. 
+
```
Comment (vaughnbetz) [inline doc/src/vpr/debug_aids.rst:125]: location you *want to move it to.* (delete would)

File: doc/src/vpr/debug_aids.rst (hunk 2 above)
Comment (vaughnbetz) [reply doc/src/vpr/debug_aids.rst:129]: Should explain what happens if a move isn't legal.

File: doc/src/vpr/debug_aids.rst [hunk 4]
```diff
@@ -103,3 +103,42 @@ Available Operators
...
+.. image:: https://www.verilogtorouting.org/img/moveCosts.png
+    :align: center
+
+If you want to make another manual move simply press proceed again and repeat the process, otherwise disable the manual move feature and continue.
+
+Selecting the block to move
+~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+You can choose the block you want to move by entering its block id or block name in the corresponding entry field, or you can simply click on that specific block. For better visualization, once you indicate your block through the entry field and press proceed, your block will be highlighted on the GUI.
```
Comment (vaughnbetz) [inline doc/src/vpr/debug_aids.rst:139]: Explain what happens if the block is part of a placement_macro (commonly due to carry chains).  Does the move get expanded to encompass any other blocks that need to move simultaneously?

Also, if you can move multiple blocks you should mention that (if you can't, don't mention it).

**Comment by vaughnbetz** [comment]
Another feature that would be useful (but may not have time for it): re-timing analyze (if the user hits a button) and show the updated critical path delay if the move was made.  Would involve interaction with Bill's new code, so you'd have to talk to him about how to make this work, and if there isn't time to try it that's OK.

File: doc/src/vpr/debug_aids.rst (hunk 1 above)
Comment (MahshadJ) [reply doc/src/vpr/debug_aids.rst]: resolved

File: doc/src/vpr/debug_aids.rst (hunk 2 above)
Comment (MahshadJ) [reply doc/src/vpr/debug_aids.rst]: resolved

File: doc/src/vpr/debug_aids.rst (hunk 3 above)
Comment (MahshadJ) [reply doc/src/vpr/debug_aids.rst:125]: resolved

File: doc/src/vpr/debug_aids.rst (hunk 4 above)
Comment (MahshadJ) [reply doc/src/vpr/debug_aids.rst]: resolved

**Comment by vaughnbetz** [comment]
Assigning to Mohamed to complete this.

**Comment by vaughnbetz** [comment]
Completed and merged by @PaulaPerdomo .  Closing.

**Pull request closed without merging**